configs/relengworker-prod/config.yml

```diff
@@ -79,7 +79,7 @@ worker_types:
     autoscale:
       algorithm: sla
       args:
-        max_replicas: 45
+        max_replicas: 25
```

Comment:
Reducing the number to deal with https://bugzilla.mozilla.org/show_bug.cgi?id=1580054#c7 and https://bugzilla.mozilla.org/show_bug.cgi?id=1591373